src/python/deepgnn/graph_engine/data/ppi.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, output_dir: Optional[str] = None, num_partitions: int = 1):
         if self.output_dir is None:
             self.output_dir = os.path.join(tempfile.gettempdir(), self.GRAPH_NAME)
         self._build_graph(self.output_dir)
-        super().__init__(path=self.output_dir, partitions=[0])
+        super().__init__(path=self.output_dir, partitions=list(range(num_partitions)))
```

Comment:
I think a lot of small stuff like this belongs in a separate PR, there are quite a few things like this.